sequencer/src/rpc.rs

```diff
@@ -176,29 +180,31 @@ impl RpcServer for SequencerServer {
     async fn get_results(
         &self, query: GetResultsQuery,
     ) -> Result<(Vec<bool>, Vec<ScoreEntry>), ErrorObjectOwned> {
-        self.db.refresh().map_err(|e| {
+        let db_handler = self.db.clone();
+        db_handler.lock().await.refresh().map_err(|e| {
```

Comment:
We dont need to do refresh inside SequencerServer, since we are doing that on interval already